### ISSUE TITLE
Can't import notificationStack

### ISSUE THREAD

**Issue by Itdotaer** [pburtchaell/react-notification#44]
Hi, why I can't import 'notificationStack' to my projects when using 
```var NotficationiStack = require('notificationStack');```. I can only use this component like this:

```js
var NotificationStack = require('../../node_modules/react-notification/dist/notificationStack');
```

And I see your projects's package.json file, found that you mainly import "dist/notification.js" to this npm module.

I advice you can sort all the Notification components to an obj and then export, and then we can using the module like this:

```js
var Notification = require('react-notification');
var SingleNotification = Notification.SingleNotification;
var NotificationStack = Notification.NotificationStack;
```

**Comment by pburtchaell** [comment]
Sorry about this issue! It slipped through the cracks when I did the release... I'll get a patch pushed out over the next few days. I agree exporting an object works best. :smile:

**Comment by pburtchaell** [comment]
This is resolved with the release of [version `4.1.1`](https://github.com/pburtchaell/react-notification/releases/tag/4.1.1).
